src/api/spec/helpers/webui/package_helper_spec.rb

```diff
@@ -253,4 +253,25 @@
     it { expect(uploadable?('image.vhdfixed.xz', 'i386')).to be_falsy }
     it { expect(uploadable?('apache2.rpm', 'x86_64')).to be_falsy }
   end
+
+  describe '#expand_diff?' do
+    it { expect(expand_diff?('ctris.spec', 'added')).to be true }
+    it { expect(expand_diff?('_patchinfo', 'added')).to be true }
+    it { expect(expand_diff?('ctris.changes', 'added')).to be true }
+    it { expect(expand_diff?('ctris.changes', 'deleted')).to be false }
+    it { expect(expand_diff?('/foo/bar/test.txt', 'added')).to be false }
```

Comment:
:thinking: I would use `be_true` and `be_false` in the last 5 lines

Reply:
``
expected true to respond to `true?` or perhaps you meant `be true` or `be_truthy`
``

https://eddyluten.com/rspec-be_truthy-exists-or-be-true

Reply:
I meant this:
https://relishapp.com/rspec/rspec-expectations/v/2-2/docs/matchers/be-matchers
What do you think?

Reply:
Oh, I see! You need to check exactly `true` and `false`. So, forget my comment!